assets/app/views/builds.html

```diff
@@ -100,7 +100,7 @@
                         <span ng-if="build.spec.source.type == 'None'">
                           <i>none</i>
                         </span>
-                        <span ng-if="build.spec.source.type == 'Git'" ng-bind-html='build.spec.source.git.uri | githubLink : build.spec.source.git.ref : build.spec.source.contextDir | linky'></span>
+                        <span class="word-break" ng-if="build.spec.source.type == 'Git'" ng-bind-html='build.spec.source.git.uri | githubLink : build.spec.source.git.ref : build.spec.source.contextDir | linky'></span>
```

Comment:
@spadgett @sg00dwin so I've tested on firefox and I couldn't reproduce the issue even without the `word-break` class. 
![screenshot-7](https://cloud.githubusercontent.com/assets/1668218/15068823/249ef48c-137a-11e6-8227-34bcf09d94ac.png)

But without the class the URL will overflow in chrome.